src/utils.ts

```diff
@@ -153,34 +153,33 @@ export const renameFilesForce = (paths: FromTo[]) =>
     tree.delete(from);
   });
 
-export function createEmptyNsOnlyProject(projectName: string, extension: string = ''): UnitTestTree {
-  let appTree = schematicRunner.runSchematic("angular-json", { name: projectName, sourceRoot: "src" });
+export function createEmptyNsOnlyProject(projectName: string): UnitTestTree {
```

Comment:
I think you removed the `extension` argument by accident. Can you get it back? It's used a few lines below.